Felienne/hedy
# [UI IMPROVEMENT] Adds close button for achievements pop-up

**Description**
In this PR we make a small improvement to the achievements pop-up, enabling users to close it by clicking the small close button at the top-right corner. 

**Fixes**
This PR fixes #2716.

**How to test**
Empty your local database Achievements table (or make sure you have an easy achievement not yet achieved). Get an achievement and verify that the closing button at the top-right corner is working as expected.
